docs/manage/users-remote.rst

```diff
@@ -0,0 +1,154 @@
+include and synchronize any information about the user stored in your IdP such as their username,
+display name, and groups. Once configured, you can manage remote users without having to manually
+modify the users or update them via SCIM. Each time the remote user accesses Determined, their
+information is synchronized.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+.. note::
+
+   As of 026.4, OIDC is supported.
+
+*******************************
+ Enable Remote User Management
+*******************************
+
+Set the Auto Provision Option
+=============================
+
+The first step is to configure the :ref:`master configuration file <master-config-reference>` to
+enable auto provisioning users and the remote management of any information attached to the users.
+
+.. tabs::
+
+   .. tab::
+
+      OIDC
+
+      -  Enable OIDC.
+      -  Set ``oidc.auto_provision_users`` option to ``true`` in your :ref:`master configuration
+         file <master-config-reference>`.
+
+      .. note::
+
+         If ``scim_enabled`` is ``true``, then ``oidc.auto_provision_users`` must be ``false``.
+
+      For example:
+
+      .. code:: yaml
+
+         oidc:
+             enabled: true
+             provider: "Okta"
+             idp_recipient_url: "https://determined.example.com/oidc/sso"
+             idp_sso_url: "https://dev-00000000.okta.com"
+             client_id: "xx0xx0"
+             client_secret: "xx0xx0"
+             auto_provision_users: true
+             display_name_claim_name: "XYZ"
+
+   .. tab::
+
+      SAML
+
+      -  Enable SAML.
+      -  Set ``saml.auto_provision_users`` option to ``true`` in your :ref:`master configuration
+         file <master-config-reference>`.
+
+      .. note::
+
+         If ``scim_enabled`` is ``true``, then ``saml.auto_provision_users`` must be ``false``.
+
+      For example:
+
+      .. code:: yaml
+
+         saml:
+             enabled: true
+             provider: "Okta"
+             idp_recipient_url: "https://determined.example.com/saml/sso"
+             idp_sso_url: "https://myorg.okta.com/app/...sso/saml"
+             idp_cert_path: "okta.cert"
+             auto_provision_users: true
+             groups_attribute_name: "groups"
+             display_name_attribute_name: "disp_name"
+         security:
+            rbac_ui_enabled: true
+            type: rbac
+
+Determined sets the username of the user to the IdP email address. You cannot set the username
+independently.
+
+Set the Groups Claim Name Option
+================================
+
+Determined receives OIDC and SAML claims via a JSON Web Token (JWT) that you send from your IdP. If
+there is a group that does not already exist in Determined, then the system creates the group.
+
+To enable group membership synchronization:
+
+-  Set the ``groups_claim_name`` option to match the claim name for group memberships from your
```

Comment:
The option is called `groups_claim_name` in OIDC, but `groups_attribute_name` in SAML

Reply:
thank you!